src/ring_theory/subsemiring/pointwise.lean

```diff
@@ -51,6 +51,10 @@ lemma pointwise_smul_def {a : M} (S : subsemiring R) :
 lemma smul_mem_pointwise_smul (m : M) (r : R) (S : subsemiring R) : r ∈ S → m • r ∈ m • S :=
 (set.smul_mem_smul_set : _ → _ ∈ m • (S : set R))
 
+lemma mem_smul_pointwise_iff_exists (m : M) (r : R) (S : subsemiring R) :
+  r ∈ m • S ↔ ∃ (s : R), s ∈ S ∧ m • s = r :=
+by { erw set.mem_smul_set, refl }
```

Comment:
Does this work?
```suggestion
(set.mem_smul_set : _ ∈ m • (S : set R) ↔ _)
```

Reply:
Yes, I have changed all of them to this.